cypress/support/commonCommands.js

```diff
@@ -9,5 +9,6 @@ Cypress.Commands.add('login', (username, password) => {
     cy.get(userNameInput).clear().type(username);
     cy.get(userPasswordInput).clear().type(password);
     cy.get(loginButton).click();
+    cy.wait(1000)
```

Comment:
Is this required? Cypress will wait for page load by default. And we have additional check in next line to get the main page header.

Reply:
Default timeout is 4 seconds. It might be because of delayed response from connection. Lets keep it then.